### PR TITLE
store/tikv/oracle: prometheus metrics for tsFuture wait time

### PR DESCRIPTION
**Pull request by tiancaiamao**
Most time it should not be zero, which means wait for a while, because Parse() & Compile() is slower than GetTS() in normal cases.
When `Wait()` duration is too small, something maybe wrong with Parse() & Compile(), or the runtime, add this metric so we can know that.

### CODE REVIEW

**Comment by tiancaiamao** [comment]
@disksing

File: store/tikv/oracle/oracles/metrics.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+// limitations under the License.
+
+package oracles
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	tsFutureWaitDuration = prometheus.NewHistogram(
```
Comment (disksing) [inline store/tikv/oracle/oracles/metrics.go]: Not registered?

Comment (tiancaiamao) [reply store/tikv/oracle/oracles/metrics.go]: Oops!

**Comment by coocood** [comment]
LGTM

**Comment by disksing** [comment]
LGTM.

**Pull request merged**